### ISSUE TITLE
Review UI for accessibility issues 

### ISSUE THREAD

**Issue by lucasgustaf** [philjng/find-my-team#32]
Submit comment button has black text on blue background. Review the rest of the app for colour contrast issues or any other accessibility issues.
